keyboards/crkbd/keymaps/madhatter/keymap.c

```diff
@@ -120,7 +117,7 @@ void update_tri_layer_RGB(uint8_t layer1, uint8_t layer2, uint8_t layer3) {
 
 void matrix_init_user(void) {
     #ifdef RGBLIGHT_ENABLE
-      RGB_current_mode = rgblight_get_mode();
+      RGB_current_mode = rgblight_config.mode;
```

Comment:
Please revert.  The preferred method is to use the query function.
```suggestion
      RGB_current_mode = rgblight_get_mode();
```